modules/database/audit-autoloaded-options/update-autoload.js

```diff
@@ -0,0 +1,45 @@
+/* eslint camelcase: "off" */
+/* global jQuery:false, perflabAutoloadSettings:false */
+
+( function ( $, document ) {
+	$( document ).on( 'click', '.update-autoload', function ( e ) {
+		e.preventDefault();
+		const button = $( this );
+		const optionName = button.data( 'option-name' );
+		const autoload = button.data( 'autoload' );
+		const value = button.data( 'value' );
+		const data = {
+			option_name: optionName,
+			autoload,
+			value,
+		};
+
+		$.ajax( {
+			url:
+				perflabAutoloadSettings.root + 'perflab-aao/v1/update-autoload',
+			method: 'POST',
+			data,
+			success( response ) {
+				if ( response.success ) {
+					button.attr( 'disabled', true );
+				} else {
+					button.append(
+						'<span class=""> ' + response.message + '</span>'
+					);
+					$(
+						'<div style="margin-top:5px;border:1px solid #d63638;padding-left:5px;">' +
+							response.message +
+							'</div>'
+					).insertAfter( button );
+				}
```

Comment:
It doesn't seem like the `else` condition ever occurs, since a non-200 status code is always returned for errors along with `success => false`. In fact, there's no need for sending `success` at all as the HTTP status can be relied on entirely.

Also, `attr()` isn't quite right here. It should be `prop()` here, although this only works with an actual `button` and not an `a` element.

```suggestion
				button.prop( 'disabled', true );
```

See https://api.jquery.com/prop/#:~:text=Attributes%20vs.%20Properties